misc/systemd/ipfs.service

```diff
@@ -3,9 +3,49 @@ Description=InterPlanetary File System (IPFS) daemon
 After=network.target
 
 [Service]
+ReadWritePaths="/var/lib/ipfs/"
+NoNewPrivileges=true
+ProtectSystem=strict
+ProtectKernelTunables=true
+ProtectKernelModules=true
+ProtectKernelLogs=true
+PrivateDevices=true
+DevicePolicy=closed
+ProtectControlGroups=true
+RestrictAddressFamilies=AF_UNIX AF_INET AF_INET6 AF_NETLINK
+ProtectHostname=true
+PrivateTmp=true
+ProtectClock=true
+LockPersonality=true
+RestrictNamespaces=true
+RestrictRealtime=true
+MemoryDenyWriteExecute=true
+SystemCallArchitectures=native
+SystemCallFilter=@system-service
+SystemCallFilter=~@privileged
+ProtectHome=true
+RemoveIPC=true
+RestrictSUIDSGID=true
+CapabilityBoundingSet=CAP_NET_BIND_SERVICE
 
+# disable for 1-1024 port listening and fuse-mount-feature
+PrivateUsers=true
 # enable for 1-1024 port listening
 #AmbientCapabilities=CAP_NET_BIND_SERVICE 
+
+# if you need the fuse-mount-feature, remove the standard security
+# settings above and use the reduced set below
+#ReadWritePaths="/var/lib/ipfs/" "/ipfs" "/ipns"
+#ProtectSystem=strict
+#ProtectControlGroups=true
+#PrivateTmp=true
```

Comment:
This will break the mount feature by running the daemon in a private mount namespace.

Reply:
Oh, right... :/